raft/raft.go

```diff
@@ -864,14 +870,14 @@ func (r *raft) loadState(state pb.HardState) {
 }
 
 // isElectionTimeout returns true if r.electionElapsed is greater than the
```

Comment:
While you're in here: how about renaming this method from `isElectionTimeout` to `pastElectionTimeout`? I think that'd be more informative.